.circleci/real_config.yml

```diff
@@ -2781,7 +2779,6 @@ workflows:
   test-go:
     jobs:
       # https://determined-ai.slack.com/archives/C03H5KZPU30/p1667241463073989
```

Comment:
Delete this as well -- it's simply a link to a message about why this test was commented out.